apps/verticalhome/js/app.js

```diff
@@ -141,6 +156,38 @@
       this.grid.stop();
     },
 
+    getAppByURL: function(url) {
+      return this.itemStore.getAppByURL(url);
+    },
+
+    getPinAppList: function() {
+      return this.itemStore.getPinAppList();
+    },
+
+    savePinAppItem: function(obj) {
+      this.itemStore.savePinAppItem(obj);
+    },
+
+    savePinApps: function(objs) {
+      this.itemStore.savePinApps(objs);
+    },
+
+    showMoreApps: function() {
+      this.inMoreApps = true;
+      console.log("inMoreApps is : " + this.inMoreApps);
+      document.getElementById("main-screen").setAttribute("hidden","");
+      document.getElementById("more-apps-screen").removeAttribute("hidden");
+      MoreAppsNavigation.init();
+      document.getElementsByTagName('gaia-grid-rs')[0].scrollTo(0,0);
+    },
+
+    backToMainScreen: function() {
```

Comment:
To be parallel with the function above, I'd suggest you call this hideMoreApps()